src/python/WMCore/MicroService/MSPileup/DataStructs/MSPileupObj.py

```diff
@@ -0,0 +1,139 @@
+"""
+File       : MSPileupObj.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: MSPileupObj module provides MSPileup data structure:
+
+{
+    "pileupName": string with the pileup dataset
+    "pileupType": string with a constant value
+    "insertTime": int, seconds since epoch in GMT timezone
+    "lastUpdateTime": int, seconds since epoch in GMT timezone
+    "expectedRSEs": ["Disk1", "Disk2", etc],  # these values need to be validated against Rucio
+    "currentRSEs": ["Disk1", "Disk3"],  # values provided by the MS itself
+    "fullReplicas": integer,  # total number of replicas to keep on Disk
+    "campaignList": [ "name", ... ] # list of workflow campaigns using this pileup
+    "containerFraction": real number with the container fraction to be distributed (TBFD)
+    "replicationGrouping": string with a constant value (DATASET or ALL, to be in sync with Rucio)
+    "activatedOn": int, seconds since epoch in GMT timezone
+    "deactivatedOn": int, seconds since epoch in GMT timezone
+    "active": boolean,
+    "pileupSizeGB": integer, current size of the pileup in GB (integer)
+    "rulesList: list of strings (rules) used to lock the pileup id
+}
+
+The data flow should be done via list of objects, e.g.
+[<pileup object 1>, <pileup object 2>, ..., <pileup object n>]
+"""
+
+# system modules
+import time
+import json
+
+# WMCore modules
+from WMCore.MicroService.Tools.Common import getMSLogger
+
+
+def gmtimeSeconds():
```

Comment:
Apologies for not mentioning this before, but the best place to have such function would be in this Utils module:
https://github.com/dmwm/WMCore/blob/master/src/python/Utils/Timers.py

Please make sure there is a relevant unit test for it.

Reply:
I'm not sure what should be relevant unit test in this case though since we only get current seconds in GMT time zone from default python library call.

Reply:
Hmm, indeed. Maybe something very basic like:
```
t1 = gmtimeSeconds()
time.sleep(1)
t2 = gmtimeSeconds()
self.assertTrue(isinstance(t2, int))
self.assertTrue(t2 > t1)
```
?

Reply:
done